core/server/worker/src/main/java/alluxio/worker/page/PagedBlockStore.java

```diff
@@ -197,13 +197,28 @@ private void commitBlockToMaster(PagedBlockMeta blockMeta) {
     }
     BlockStoreLocation blockLocation =
         new BlockStoreLocation(DEFAULT_TIER, getDirIndexOfBlock(blockId));
+
+    commitLocalEvent(blockId, blockLocation);
+  }
+
+  public void commitLocalEvent(long blockId, BlockStoreLocation location) {
```

Comment:
this is a bad naming...
you can either not to use a method, or rename this to triggerOnBlockCommitttedToLocalEvent